cmd/osbuild-worker/jobimpl-osbuild.go

```diff
@@ -88,6 +88,8 @@ func (impl *OSBuildJobImpl) Run(job worker.Job) error {
 
 	end_time := time.Now()
 
+	var streamOptimizedPath string = ""
```

Comment:
This can be simplified as:

```suggestion
	streamOptimizedPath := ""
```

(no need to fix this though, I'm fine with the longer form)